components/server/src/workspace/workspace-starter.ts

```diff
@@ -836,7 +836,12 @@ export class WorkspaceStarter {
             await this.tryEnableConnectionLimiting(featureFlags, user, billingTier);
             await this.tryEnablePSI(featureFlags, user, billingTier);
 
-            const usageAttributionId = await this.userService.getWorkspaceUsageAttributionId(user, workspace.projectId);
+            let usageAttributionId = await this.userService.getWorkspaceUsageAttributionId(user, workspace.projectId);
+            // if the workspace has been created in an organization, we need to use the organization's attribution ID
+            if (workspace.organizationId) {
+                const org = await this.teamDB.findTeamById(workspace.organizationId);
+                usageAttributionId = AttributionId.create(org!);
+            }
```

Comment:
Nice. Would love to be in a position where all workspaces have an Org ID, such that we don't need to do these extra checks. We'll get there soon enough.